frontend/sw360-portlet/src/main/webapp/html/admin/fossyAdmin/view.jsp

```diff
@@ -57,9 +58,22 @@
     </core_rt:if>
 </form>
 
+<core_rt:set var="fossologyEnabled" value="<%=FossologyConnectionHelper.FOSSOLOGY_CONNECTION_ENABLED%>"/>
+
+<core_rt:choose>
+	<core_rt:when test="${fossologyEnabled}">
+	    <core_rt:set var="currentConnStatus" value="Enabled" />
+	</core_rt:when>
+	<core_rt:otherwise>
+		<core_rt:set var="currentConnStatus" value="Disabled" />
+	</core_rt:otherwise>
+</core_rt:choose>
+
 <button class="addButton" onclick="deployScripts('deployResult')">Deploy Scripts To Fossology Server</button>
 <span id="deployResult"></span>
 
+<div>Fossology Connection Status: <span id="conStatus"><b>${currentConnStatus}</b></span></div>
```

Comment:
I think this page has not been a beauty before, but wouldn't it be better to place button and this text together (maybe in own block with heading)? And I am not sure if we need the existing output next to the button that pops up e.g. with "Failure" if the connection is not available.